charts/prometheus-redis-exporter/templates/podsecuritypolicy.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.rbac.pspEnabled }}
+{{ if $.Capabilities.APIVersions.Has "policy/v1/PodSecurityPolicy" -}}
+apiVersion: policy/v1
+{{- else -}}
 apiVersion: policy/v1beta1
+{{- end }}
```

Comment:
Sorry I'm closing this since I realized that API doesn't even exist, `PodSecurityPolicy` is completely deprecated and will be replaced by `Pod Security Admission`.